src/renderer/views/wallet/NoWalletView/NoWalletView.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useMemo } from 'react'
+
+import Tabs from '../../../components/Tabs'
+import { MainnetView } from './MainnetView'
+
+enum TabKey {
+  TESTNET = 'testnet',
+  MAINNET = 'mainnet'
+}
+
+type Tab = {
+  key: TabKey
+  label: string
+  content: React.ReactNode
+}
+
+const NoWalletView = () => {
+  const items: Tab[] = useMemo(
+    () => [
+      { key: TabKey.TESTNET, label: 'testnet', content: <span>testnet</span> },
+      { key: TabKey.MAINNET, label: 'mainnet', content: <MainnetView /> }
+    ],
+    []
+  )
+
+  return <Tabs centerContent={true} tabs={items} defaultTabIndex={1} />
```

Comment:
_Just FYI (you can't know it and we have not addressed it in FIGMA yet):_ We don't need tabs to make a selection for test- or mainnet to import a wallet. Because same keystore can be used in test- and mainnet. 

See following [question / answer in our Slack history](https://thorchain.slack.com/archives/C012CLYR1DG/p1592898922420100):
Q: (4) Can an user use a same wallet (keystore) for mainnet and testnet?
A: 4) yes this is no problem. Keystores work fine for any network